manuscript/text/01_introduction.md

```diff
@@ -25,28 +25,44 @@ affiliation:
+In 2018, over 1.7 million new cases of cancer were estimated to be diagnosed with over 609,000 of those cases projected to be fatal.[@Smith; @] One of the primary treatments of cancer consists of chemotherapy, particularly targeted therapies, whereby patients are given drugs that destroy highly-prolific cells to stall cancer growth or eliminate the tumor. However, long-term therapeutic efficacy varies significantly due to the vast heterogeneity in intratumor response to therapy.[@Di Maio; @De Roock] Specifically, each tumor consists of cancer cell subpopulations that differ in terms of cell intrinsic and extrinsic factors including genetic plasticity, epigenetic alterations, and micro-environmental stressors.[@Semenza; @Nagarsheth; @Feinberg; @Falkenberg] Current drug-screening protocols involve giving known cancer cell lines different drug doses and evaluating therapy performance based on the percent of cells eliminated and the dose required for half-maximal effect.[@Chauvin; @Zhang_L] These metrics, however, provide population averages of overall tumor response to the therapy and fail to consider the subpopulation heterogeneity. In addition, there is a need for real-time cell classification, which would identify the significant proportion of cells that undergo stochastic changes in resistance after onset of therapy.[@Gupta; @Enderling; @Inde]
```

Comment:
Are you sure spaces can be in the cite-key?

Reply:
No they can't, pretty sure they're the reason for failure.

Reply:
Fixed.